README.md

```diff
@@ -11,15 +11,24 @@ A guide for front-end development at the [CFPB](http://cfpb.github.io/).
 - [CSS/Less standards](css.md)
 - [JavaScript standards](javascript.md)
 
-## Guidelines
+## Capital Framework
+
+Capital Framework is our modular front end framework.
+
+- [Capital Framework docs](http://cfpb.github.io/capital-framework/)
+- [cf-generator](https://github.com/cfpb/generator-cf)
+
+## Guides
 
 - [Publishing node modules](npm.md)
 - [Building your project's front-end](build.md)
+- [Code review guide](code-reviews.md)
 
 ## Tools
 
 - [ESLint file](.eslintrc)
 - [Browser testing checklist](browser-checklist.md)
+- [Pull request template](https://raw.githubusercontent.com/cfpb/front-end/master/pr-template.md)
```

Comment:
I intentionally linked to the raw document so that it can be easily copy and pasted. Is there a way to do that with a relative link?